addons/analytic/static/src/components/analytic_distribution/analytic_distribution.xml

```diff
@@ -25,7 +25,7 @@
                     </div>
                 </div>
             </div>
-            <div class="popover-body p-2 table-responsive">
+            <div class="popover-body p-2 table-responsive" style="overflow-x: auto; max-width: 100vw;">
```

Comment:
```suggestion
            <div class="popover-body p-2 table-responsive" style="max-width: 100vw;">
```

no need to add `overflow-x: auto;` style as it's already covered in `table-responsive` class